www/src/pages/components/modal-curtain/react/index.mdx

```diff
@@ -11,164 +11,107 @@ import { ComponentHeader, ComponentFooter } from 'components/thumbprint-componen
 
 ## Examples
 
-The `ModalCurtain` component makes it easy to create an accessible overlay that contains content and covers the page. It contains a few powerful features:
+The `ModalCurtain` component makes it easy to create an accessible overlay that contains content and covers the page.
```

Comment:
Can we not change these in future PRs? Makes the PRs noisier and the style inconsistent with the rest of the codebase.